store-implementation/federated-store/src/test/java/uk/gov/gchq/gaffer/federatedstore/FederatedStoreToFederatedStoreTest.java

```diff
@@ -61,61 +62,64 @@
     @BeforeEach
     public void setUpStores() throws OperationException {
         SingleUseFederatedStore.cleanUp();
+        CacheServiceLoader.shutdown();
 
         ProxyProperties proxyProperties = new ProxyProperties();
         proxyProperties.setStoreClass(SingleUseFederatedStore.class);
 
         restApiFederatedGraph = new Graph.Builder()
-            .storeProperties(proxyProperties)
-            .config(new GraphConfig("RestApiGraph"))
-            .addSchema(new Schema())
-            .build();
+                .storeProperties(proxyProperties)
+                .config(new GraphConfig("RestApiGraph"))
+                .addSchema(new Schema())
+                .build();
 
+        FederatedStoreProperties properties = new FederatedStoreProperties();
+        properties.setCacheServiceNameSuffix("Alt");
         federatedStoreGraph = new Graph.Builder()
-            .config(new GraphConfig("federatedStoreGraph"))
-            .storeProperties(new FederatedStoreProperties())
-            .build();
+                .config(new GraphConfig("federatedStoreGraph"))
+                .storeProperties(properties)
+                .build();
 
         connectGraphs();
         addMapStore();
     }
 
     private void addMapStore() throws OperationException {
         restApiFederatedGraph.execute(new AddGraph.Builder()
-            .storeProperties(new MapStoreProperties())
-            .graphId("mapStore")
-            .schema(Schema.fromJson(getClass().getResourceAsStream("/schema/basicEntitySchema.json")))
-            .build(), new User());
+                .storeProperties(new MapStoreProperties())
+                .graphId("mapStore")
+                .schema(Schema.fromJson(getClass().getResourceAsStream("/schema/basicEntitySchema.json")))
+                .build(), new User());
     }
 
     private void connectGraphs() throws OperationException {
         federatedStoreGraph.execute(new AddGraph.Builder()
-            .storeProperties(new ProxyProperties())
-            .graphId("RestProxy")
-            .schema(new Schema())
-            .build(), new User());
+                .storeProperties(new ProxyProperties())
+                .graphId("RestProxy")
+                .schema(new Schema())
+                .build(), new User());
     }
 
     @Test
     public void shouldErrorIfViewIsInvalid() throws OperationException {
         // Given
         Entity entity = new Entity.Builder()
-            .group(BASIC_ENTITY)
-            .vertex("myVertex")
```

Comment:
Why have these been changed? Constants can be made for `myVertex`, `mySource`, `myDest` and `property1` perhaps?

Reply:
Unsure, it may have been a relic left behind from debugging.